### PR TITLE
Mentors hotfix 3

### PR DESCRIPTION
**Pull request by RyanRTJJ**
New csv and new json to include all changes labeled 'hotfix #3' on the google doc

### CODE REVIEW

File: src/mentors.json [hunk 1]
```diff
@@ -506,8 +506,8 @@
...
...t deploys proprietary machine learning and natural language processing algorithms. She graduated from the Nanyang Technological University with a bachelor's degree in business administration, after which she took up a job as a Customer Affairs Manager with Singapore Airlines\u00a0\u2014\u00a0this was not her first job, though, for her foray into the working world began as early as the tender age of 15. Since working in customer success then, Felicia has gone on to hold appointments in Business Partnerships for Singapore Airlines, and Public Service Leadership at the Public Service Division. In the course of her work, however, she has undertaken a whole\u2014wider\u2014range of portfolios, from crisis management to policy-making and people operations, spanning a diversity of industries, such as aviation, government and technology. To Felicia, she observes the common thread in her professional experiences to be bringing people with different perspectives together to solve challenging problems. This motivates her, as she relishes the opportunity to help others maximise their growth potential for greater impact, and learn from the expertise and experiences of others, such as her own.",
```
Comment (wei2912) [inline src/mentors.json:513]: Biography should be changed to "INTELLLEX" too (three 'L's).

File: src/mentors.json [hunk 2]
```diff
@@ -1595,11 +1595,11 @@
...
       "imageUrl": "/profile_images/SereneGoh.jpg"
```
Comment (wei2912) [inline src/mentors.json:1603]: Serene's profile picture seems to be removed. The new profile picture shld be uploaded instead, see the spreadsheet.

File: src/mentors.json [hunk 3]
```diff
@@ -636,7 +636,7 @@
...
+      "fullBio": "Hemaa Sekar is an Analyst at Vriens & Partners, Singapore whose role is to provide effective and nuancedHemaa Sekar is an Analyst at Vriens & Partners, Singapore. As a junior government affairs consultant, she supports on projects spanning several sectors, particularly ICT, transport and healthcare, to provide effective and nuanced public policy and government affairs counsel. She served as Legislative Assistant at the Parliament of Singapore to MP Louis Ng for a year, drafting parliamentary speeches and questions. She has also interned at the US-ASEAN Business Council and UN Women Singapore. Hemaa graduated from NUS with an honours degree, double majoring in Political Science and Economics and is a Dean's List recipient.",
```
Comment (wei2912) [inline src/mentors.json:639]: Looks like there's a copy paste error here.

File: src/mentors.json (hunk 1 above)
Comment (RyanRTJJ) [reply src/mentors.json:513]: Thanks for catching this

File: src/mentors.json (hunk 2 above)
Comment (RyanRTJJ) [reply src/mentors.json:1603]: Is it there.

Comment (wei2912) [reply src/mentors.json:1603]: Oops lol I mistook the decrease in size to be a delete.

**Pull request merged**